GherkinSyncTool.Synchronizers.TestRail/Content/CaseContentBuilder.cs

```diff
@@ -83,33 +84,8 @@ private List<CustomStepsSeparated> ConvertToCustomStepsSeparated(List<string> st
 
         private string ConvertToStringTags(Scenario scenario, IFeatureFile featureFile)
```

Comment:
Not sure it will be more readable then:
![image](https://user-images.githubusercontent.com/24970976/141251613-dc9eaab1-9ea1-45b5-8caf-7c80c5c6c028.png)